tests/test_cli.py

```diff
@@ -129,3 +129,11 @@ def test_remove_nonexistent(self):
     @pytest.mark.parametrize("repo", TEST_REPOS)
     def test_list(self, repo):
         self._list(repo)
+
+    def test_triton_profile(self, mocker, monkeypatch):
+        test_args = ["triton", "profile", "-m", "add_sub"]
+        mock_run = mocker.patch("subprocess.run")
+        monkeypatch.setattr("sys.argv", test_args)
+        args = parse_args()
+        args.func(args)
+        mock_run.assert_called_once_with(["genai-perf", "-m", "add_sub"], check=True)
```

Comment:
Can you ELI5 or add a comment on what this test is doing? Is it just asserting that the command gets transformed to the right genai-perf command correctly?

Reply:
Yes, exactly. :) It mocks the subprocess call to make sure the right subprocess call gets made once (but not actually executed). I added a comment.